Description: Expand the GtkTreeView. Otherwise set_cursor() will fail, 
 making get_cursor() return NULL, making to_string() error out with
 .
 Can't call method "to_string" on an undefined value at t/GtkTreeView.t 
 line 233.
Origin: vendor, http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=22;filename=gtktreeview_expand;att=1;bug=545616
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=49831
Bug-Debian: http://bugs.debian.org/545616
Author:Kristian Rietveld and Daniel Leidert <daniel.leidert@wgdd.de>
Forwarded: yes

--- a/t/GtkTreeView.t
+++ b/t/GtkTreeView.t
@@ -229,6 +229,7 @@
 isa_ok($view -> get_background_area(undef, $view_column_two), "Gtk2::Gdk::Rectangle");
 isa_ok($view -> get_background_area($path, undef), "Gtk2::Gdk::Rectangle");
 
+$view -> expand_all();
 $view -> set_cursor(Gtk2::TreePath -> new("1:0"), $view_column_one, 0);
 is(($view -> get_cursor())[0] -> to_string(), "1:0");
 is(($view -> get_cursor())[1], $view_column_one);
@@ -614,6 +615,7 @@
 	skip("set_cursor_on_cell is new in 2.2.x", 2)
 		unless Gtk2->CHECK_VERSION (2, 2, 0);
 
+	$view -> expand_all();
 	$view -> set_cursor_on_cell(Gtk2::TreePath -> new("1:1"),
 				    $view_column,
 				    $cell_renderer,
